yarn.lock

```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+castle.js@^1.0.21:
+  version "1.0.21"
+  resolved "https://registry.yarnpkg.com/castle.js/-/castle.js-1.0.21.tgz#bcdc90f64152d666564151481cc60f2d890148e3"
```

Comment:
What additional data is being sent from the js library? Would it possible to use CSP with this?